numba/targets/tupleobj.py

```diff
@@ -188,6 +188,8 @@ def iternext_unituple(context, builder, sig, args, result):
                                        types.intp)
         getitem_out = getitem_unituple(context, builder, getitem_sig,
                                        [tup, idx])
+        if context.enable_nrt:
```

Comment:
Want to add a comment here for future visitors of this code path?

Reply:
done